qemu/tests/balloon_check.py

```diff
@@ -95,6 +104,7 @@ def balloon_memory(self, new_mem):
             self.current_mmem = self.get_ballooned_memory()
             compare_mem = self.current_mmem
         else:
+            time.sleep(60)
```

Comment:
It applied in following environment:
  When VM balloon from 16G to 3G,  windows guest memory needs time to change to the setting value and keep stable.